_includes/references.md

```diff
@@ -44,7 +44,6 @@ for details --> {% endcomment %}
 {% comment %}<!-- Later link definitions supersede earlier definitions.
 When more recent information about a BIP is available not in the regular
 place, put links here. -->{% endcomment %}
-[BIP151]: https://gist.github.com/jonasschnelli/c530ea8421b8d0e80c51486325587c52
```

Comment:
Note: this gist previously pointed to the updated version of BIP151.  However, now its latest HEAD points to the v2 encrypted protocol, so I've dropped it so that we link to the old version of BIP151 instead.  An alternative would be for me to find the previous updated-151 version in the gist repository history (let me know if you want that; I have no preference here).